support-frontend/assets/components/list/list.jsx

```diff
@@ -31,35 +50,118 @@ const listItem = css`
 
 const listItemBullet = css`
   display: inline-block;
+  border-radius: 50%;
+  /* Sit the bullet in the vertical centre of the first line */
+  margin-top: calc((1.5em - ${space[3]}px) / 2);
+`;
+
+const listItemBulletLarge = css`
   width: ${space[3]}px;
   height: ${space[3]}px;
-  border-radius: 50%;
-  background-color: ${brandAlt[400]};
-  margin-top: ${space[1]}px;
 
   ${from.tablet} {
+    margin-top: calc((1.5em - ${space[4]}px) / 2);
     width: ${space[4]}px;
     height: ${space[4]}px;
   }
 `;
 
+const listItemBulletSmall = css`
+  width: ${space[3]}px;
+  height: ${space[3]}px;
+`;
+
+const listItemBulletLight = css`
+  background-color: ${brandAlt[400]};
+`;
+
+const listItemBulletDark = css`
+  background-color: ${brand[400]};
+`;
+
 const listItemContent = css`
-  ${body.medium()};
   margin-left: ${space[2]}px;
   max-width: 90%;
 `;
 
-function List({ items }: ListPropTypes) {
+const listItemMainText = css`
+  display: block;
+  font-weight: 700;
+`;
+
+const bulletColours: { [key: ListBulletColour]: string } = {
+  light: listItemBulletLight,
+  dark: listItemBulletDark,
+};
+
+const bulletSizes: { [key: ListBulletSize]: string } = {
+  large: listItemBulletLarge,
+  small: listItemBulletSmall,
+};
+
+function ListItem({ item, colour, size }: ListItemProps) {
   return (
-    <ul css={list}>
-      {items.map(item => (
-        <li css={listItem}>
-          <span css={listItemBullet} />
-          <span css={listItemContent}>{item.explainer}</span>
-        </li>
-    ))}
-    </ul>
+    <li key={item.explainer} css={listItem}>
+      <span css={[
+        listItemBullet,
+        bulletColours[colour],
+        bulletSizes[size],
+      ]}
+      />
+      <span css={listItemContent}>{item.explainer}</span>
```

Comment:
just to confirm - if subText is passed in it will be just ignored for this list item

Reply:
Yep, this component isn't supposed to show it